moto/secretsmanager/models.py

```diff
@@ -136,6 +136,7 @@ def to_dict(self):
             "DeletedDate": self.deleted_date,
             "Tags": self.tags,
             "VersionIdsToStages": version_id_to_stages,
```

Comment:
Does this mean this field is no longer necessary?

Reply:
They're both required.  The original one for the `describe_secret` response and the one I added for the `list_secrets` response.

Reply:
Ah, gotcha - didn't realize that this dict feeds both methods